src/MariaDb/custom/Get-AzMariaDBConnectionString.ps1

```diff
@@ -12,6 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ----------------------------------------------------------------------------------
+
+<#
+.Synopsis
+Get connection string of a MariaDb under a given framework.
```

Comment:
A small suggestion: we should use `MariaDB` in strings displayed to customer, because it's more formal. `MariaDb` should only be used internally.

Reply:
> A small suggestion: we should use `MariaDB` in strings displayed to customer, because it's more formal. `MariaDb` should only be used internally. Otherwise it LGTM

Your proposal make sense. However, currently we use MariaDb as the term of MariaDB in the Azure PowerShell

https://docs.microsoft.com/en-us/powershell/module/az.mariadb/?view=azps-4.7.0
https://docs.microsoft.com/en-us/powershell/module/az.mariadb/get-azmariadbconnectionstring?view=azps-4.7.0

We should change all of them to keep consistency. 
Can you keep the current "MariaDb" temporarily?

Reply:
As discussed, I will merge this PR and move the discussion for naming in another issue.